tests/php/test-specs.yml

```
@@ -7,7 +7,9 @@ scenarios:
       - sleep 5
       - kubectl wait --for=condition=Ready -n default --all pods --timeout=2m
       - curl --fail-with-body $(minikube service test-app-php-service --url -n default)
+      - sleep 5
       - curl --fail-with-body $(minikube service test-app-php-service --url -n default)
+      - sleep 5
```

```suggestion
      - curl --fail-with-body $(minikube service test-app-php-service --url -n default)
```
I think these sleeps are unnecessary since the test app already `sleeps` and we've identified that timing isn't the problem in these test failures.

---

You're right, I'll open the follow up PR to test PHP changes for when they release the fix and I'll trim those back out. I'd rather not dismiss all our approvals again 😅